sematic/ui/packages/main/src/pipelines/RunTabs.tsx

```diff
@@ -56,6 +57,7 @@ export default function RunTabs(props: {
             <Tab label="Logs" value="logs" />
             {grafanaTab}
             <Tab label="Resources" value="ext_res" />
+            <Tab label="Pods states" value="pod_lifecycle" />
```

Comment:
Can we potentially combine this with the "Resources" tab? We could call it something like "Infrastructure?" If not, I think "Kubernetes Debug" is maybe a better name that gets across the intent better?

Reply:
The visual style of this and the resources are not exactly the same. One has an accordion; one does not have one. Combined in one tab, they might not look good back to back. Another difference is that one actively pulls the other passively listens through WebSockets. We had an optimization: we stop active pulling when we switched away from the tab. If we keep the unified tab open to see pod lifecycles, constant pulling is unavoidable. This might degrade UX. I am open to combining, but let's have some way to address these issues before doing that. 

I can now change it to " Kubernetes Debug" as a dedicated tab.

Reply:
Keeping them in separate tabs seems fine given the performance issues. We actually don't know how we want to add subsequent resources, and putting them all together in one tab could get crowded. You would also would have 2 different timelines rendered in the same space, and you would potentially need to scroll between them, or look at both of them in different stages of their lifecycles. Let's not confuse the users.

I would prefer "Pods" or "Kubernetes" over "Kubernetes Debug". "Debug" suggests an uncommon and disruptive (non-productive) activity, and perusing or monitoring the pod states should be a common action.

Reply:
I think "Pods" might be the best option since the Resources tab also has something to do with k8s. The Ray cluster is built on top of k8s.  Mentioning 'Kubernetes' might inject ambiguity/confusion between the two tabs. "Pods" is more direct, straight, and specific to what the tab actually does. 

I adopt "Pods" for now. Let me know if any of you feel strongly, we can change otherwise. It will be an easy wording change without much effort.